.gitignore

```diff
@@ -1,2 +1,4 @@
 
 config/config.php
+.DS_Store
+17thblog/blog
```

Comment:
I could be wrong, but it looks like you're ignoring this whole directory. Is that intentional? Did you mean to delete it instead?

Reply:
I'm sorry, but do you mean that I'm ignoring 17thblog/blog and .DS_Store? I did create a file that was called "blog" instead of "blog.html" by accident, but I think I may have put it in the ignore folder (although I'm not totally sure what that ignore folder does). I also remember seeing DS_Store but I didn't know what it was/I never created it so I think I put it in the ignore folder too?

According to wikipedia, ".DS_Store is a file that stores custom attributes of its containing folder." Can I delete this DS Store file? Is DS Store it necessary? Should I take these files out of ignore?

Reply:
"Ignoring" a file means that you want to have it on your own computer, but you don't want to track it in git (i.e. it won't be uploaded to github, but it will be on your computer). `.DS_Store` needs to be ignored because Macs make it by default, so you sort need it on your computer, but you don't need it in git.

Another good use for ignoring files is that you might need to save some passwords in your code (like the password to access the database which we'll talk about soon) and you don't want that uploaded to github because it's private/sensitive.

Ignoring a file is different from deleting it. If you don't want it at all, just delete it (and that deletion can be committed into git) rather than ignoring it, because you don't want the file in github or on your local computer.